library-extensions/src/main/java/com/mikepenz/fastadapter_extensions/utilities/SubItemUtil.java

```diff
@@ -305,6 +305,54 @@ else if (!subItemsOnly && getParent(item) == null) {
         return deleted;
     }
 
+    /**
+     * notifies sub items, if they are currently extended
+     *
+     * @param adapter the adapter
+     * @param identifiers set of identifiers that should be notified
+     */
+    public static <T extends IItem & IExpandable> void notifyItemsChanged(final FastAdapter adapter, Set<Long> identifiers) {
+        int i;
+        IItem item;
```

Comment:
You mean `com.mikepenz.fastadapter.IItem`? Same comment as below applies, this functions hands on the headers (which must implement `IExpandable`) to the recursive function. So why don't force this?